eng/pipelines/common/provision.yml

```diff
@@ -79,6 +79,26 @@ steps:
       condition: and(succeeded(), eq(variables['Agent.OS'], 'Darwin'))
       timeoutInMinutes: 30
 
+  # Provision Additional Software
+  - ${{ if ne(parameters.skipProvisionator, true) }}:
+    # Prepare macOS
+    - task: xamops.azdevex.provisionator-task.provisionator@3
+      displayName: 'Provision Additional Software'
+      condition: and(succeeded(), eq(variables['Agent.OS'], 'Darwin'))
+      continueOnError: true
+      inputs:
+        provisioning_script: ${{ parameters.checkoutDirectory }}/${{ parameters.provisionatorPath }}
+        provisioning_extra_args: ${{ parameters.provisionatorExtraArguments }}
+        github_token: ${{ parameters.gitHubToken }}
+      env:
+        PROVISIONATOR_CHANNEL: ${{ parameters.provisionatorChannel }}
+        AUTH_TOKEN_COMPONENTS_MAC_IOS_CERTIFICATE_P12: ${{ parameters.certPass }}
+        ${{ if eq(parameters.skipAndroidSdks, true) }}:
```

Comment:
I think we can remove all this since this is not done on msbuild from @mattleibow  pr

Reply:
we can remove lines 96 to 101

Reply:
Removed that lines.